assignment4.Rmd

```diff
@@ -77,18 +79,26 @@ plot(pca, type = "lines")
 #Think about which components you would drop and make a decision
 
 ```{r}
+#keep first three components
+D4 <- dplyr::select(D2, 1:3)
+D4 <- scale(D4, center = TRUE)
+
 #Now, create a data frame of the transformed data from your pca.
 
-D3 <- as.data.frame(pca$x)
+D5 <- as.data.frame(pca$x)
 
 #Attach the variable "mean_correct" from your original data frame to D3.
 
-D4 <- cbind(D3, as.data.frame(D1$mean_correct))
+D5 <- cbind(D4, as.data.frame(D1$mean_correct))
 
 #Now re-run your scatterplots and correlations between the transformed data and mean_correct. If you had dropped some components would you have lost important infomation about mean_correct?
 
-COR2 <- 
+COR2 <- cor(D5)
+corrplot(COR2, order="AOE", method="circle", tl.pos="lt", type="upper",        
+tl.col="black", tl.cex=0.6, tl.srt=45, 
+        addCoef.col="black", addCoefasPercent = TRUE,
+        sig.level=0.50, insig = "blank")
 
-  
+#This loses some of the high correlations from the original (between mean_hints and mean_attempts, for example). Not sure if this was the end product I was supposed to get to? 
 ```
```

Comment:
Pretty much, I want you to demonstrate the trade off between creating a more efficient dataset (fewer variables) and the possibility that you will lose some important variation. In this case a correlation between mean correct and the sixth component.